src/solutions/solution_interface.jl

```diff
@@ -93,6 +93,8 @@ Base.@propagate_inbounds function Base.getindex(A::AbstractTimeseriesSolution, s
                 return A.t
             elseif has_sys(A.prob.f) && is_param_sym(A.prob.f.sys, sym)
                 return A.prob.p[param_sym_to_index(A.prob.f.sys, sym)]
+            elseif has_sys(A.prob.f) && Symbol(sym) in Set(Symbol(A.prob.f.sys.name,:₊,x) for x in getparamsyms(A))
```

Comment:
@YingboMa is this fine? It should use a generator, but fine with supporting that way of de-namespacing?